src/components/Fetch.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import './Fetch.css';
+
+export default function Fetch({ value, onSearchItem }) {
```

Comment:
`Fetch` is not a great name for this component. Maybe something like `SearchedShoppingList`?